File: Service_eHospital_DongNai_A/Implement/CDT_KeHoachHoiThao_CBThamGiaService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class CDT_KeHoachHoiThao_CBThamGiaService : BaseService<CDT_KeHoachHoiThao_CBThamGia, ICDT_KeHoachHoiThao_CBThamGiaRepository>
    , ICDT_KeHoachHoiThao_CBThamGiaService
    {
    private readonly ICDT_KeHoachHoiThao_CBThamGiaRepository _CDT_KeHoachHoiThao_CBThamGiaRepository;
    public CDT_KeHoachHoiThao_CBThamGiaService(ICDT_KeHoachHoiThao_CBThamGiaRepository CDT_KeHoachHoiThao_CBThamGiaRepository) : base(CDT_KeHoachHoiThao_CBThamGiaRepository)
    {
    _CDT_KeHoachHoiThao_CBThamGiaRepository = CDT_KeHoachHoiThao_CBThamGiaRepository;
    }
    }
    }

File: Report/src/app/shared/eHospital_DongNai_A/CDT_KeHoachHoiThao_CBThamGia.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { CDT_KeHoachHoiThao_CBThamGia } from './CDT_KeHoachHoiThao_CBThamGia.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class CDT_KeHoachHoiThao_CBThamGiaService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'CDT_KeHoachHoiThao_Id', 'HoiThao_Id', 'NhanVien_Id', 'BenhVien_Id', 'GhiChu', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: CDT_KeHoachHoiThao_CBThamGia[] | undefined;
    ListFilter: CDT_KeHoachHoiThao_CBThamGia[] | undefined;
    FormData!: CDT_KeHoachHoiThao_CBThamGia;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "CDT_KeHoachHoiThao_CBThamGia";
    }
}

